src/presenters/admin/indexing-notification-presenter.php

```diff
@@ -80,6 +80,12 @@ protected function get_message( $reason ) {
 			case Indexing_Notification_Integration::REASON_HOME_URL_OPTION:
 				$text = \esc_html__( 'Because of a change in your home URL setting, some of your SEO data needs to be reprocessed.', 'wordpress-seo' );
 				break;
+			case Indexing_Notification_Integration::REASON_CATEGORY_BASE_PREFIX:
```

Comment:
There's already a case for `REASON_CATEGORY_BASE_PREFIX` at line 77. I guess it should be removed.